src/css/App.css

```diff
@@ -45,11 +46,59 @@
   text-align: center;
 }
 
+.Quote-text-container {
+  height: 20rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
 .Quote-text {
   margin-left: 2rem;
   margin-right: 2rem;
   font-family: "Source Serif Pro";
   font-weight: 200;
   font-style: italic;
-  font-size: 2rem;
+  font-size: 1.75rem;
+}
+
+.Button {
+  font-size: 1.5rem;
+  font-family: "Luckiest Guy";
+  color: white;
+  width: 30%;
+  border-radius: 50px;
+  padding: 0.75rem 3rem 0.25rem 3rem;
+  margin-bottom: 1rem;
+  border: none;
+  transition-property: all;
+  transition-duration: 300ms;
+}
+
+.Button:hover {
+  cursor: pointer;
+  transition-property: all;
+  transition-duration: 200ms;
+  transform: scale(1.1);
+}
+
+.Button:focus {
+  outline: none;
+}
+
+.Button-orange {
+  background-color: rgb(243, 106, 42);
```

Comment:
Another small point and this is fine and doesn't need changing, but you'll mostly see hexadecimal values being used in projects rather than `rgb()` - hex values like `#FFFFFF` are easier to read as a developer once you understand how they work

Reply:
Ah yeah to be honest I'm much more familiar with hexadecimal values, I'm not entirely sure why these are in there as RGB - I think when I build KONK originally there was a colour picker within the editor or something and it automatically set them as RGB. But regardless, cheers - I'll make sure hexadecimal are used going forward.